exercises/change/.meta/template.j2

```diff
@@ -7,13 +7,12 @@ class {{ exercise | camel_case }}Test(unittest.TestCase):
         {%- set coins = case["input"]["coins"] %}
         {%- set target = case["input"]["target"] %}
         {%- set expected = case["expected"] %}
-        {%- if case["expected"] is mapping %}
+        {%- if case is error_case %}
         with self.assertRaisesWithMessage(ValueError):
             {{case["property"] |to_snake }}({{coins}}, {{target}})
         {% else %}
         self.assertEqual({{ case["property"] |to_snake }}({{coins}}, {{target}}), {{expected}})
         {% endif %}
     {% endfor %}
 
-{{ macros.utility() }}
-{{ macros.footer() }}
\ No newline at end of file
+{{ macros.footer(has_error_case) }}
```

Comment:
Apologies to switch on you: #2084 was just merged, and makes `has_error_case` implicit, so it is no longer required here. We can go ahead and merge this as-is, since the parameter is still supported, but keep this change in mind for future pull requests.

Reply:
KK good to know!